pkg/router/f5/f5.go

```diff
@@ -459,6 +459,35 @@ func (f5 *f5LTM) delete(url string, result interface{}) error {
 }
 
 //
+// iControl REST resource helper methods.
+//
+
+// iControlResourceUriPath returns a fully qualified `escaped` resource URI path component that can be used in the iControl REST calls.
```

Comment:
I'll update those comments - no reason really to quote escaped other than make it stand out/indicate that the path component for the iControl resource (within the URI) is escaped as '/' is a  HTTP resource path delimiter. 

I originally used `iControlResourceUriPathComponent` as the name but it felt a bit too long and awkward, but if that helps make it clearer ...